Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -1494,7 +1502,9 @@ BYE BYE BORDERS
     { color: #00b5c1; text-decoration: none; }
 
     .z-cse .x-btn-arrowslide-toolbar-small:hover
-    { background-color: #00b5c1 !important; }
+    { background-color: #b6cbe4 !important; }
```

Comment:
could you provide a GIF of the arrow icon changes here?  thanks.

Reply:
![123](https://user-images.githubusercontent.com/37290431/51253677-ad587880-19a7-11e9-943a-79bfce5e7881.gif)